agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/common/Constants.java

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.applicationinsights.agent.internal.common;
+
+public final class Constants {
```

Comment:
since these constants are only used in 2 files, maybe duplicate the constants in each? (sorry, I'm aways trying to avoid the massive centralized constants file, we could also maybe scope the class name a bit better as an alternative)

Reply:
hardcoded strings are prompt for mistake and if spec gets updated, need to update it everywhere. 
a centralized constants file is easier to maintain in my opinion. i prefer not to duplicate the constants in every class the same string is used. let's chat more tomorrow in standup @trask